build.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Looks like this is some personal stuff that snuck in.

Reply:
Good catch, cleaned it up, `git add -A` strikes again.